package.json

```diff
@@ -65,11 +70,19 @@
     "lint-staged": "^14.0.1",
     "prettier": "^3.0.3",
     "semver": "^7.5.4",
+    "stylelint": "^15.10.3",
+    "stylelint-config-property-sort-order-smacss": "^9.1.0",
+    "stylelint-config-standard-scss": "^11.0.0",
     "turbo": "^1.10.13",
     "typescript": "^5.2.2"
   },
   "packageManager": "pnpm@8.6.12",
   "engines": {
     "node": ">=18.5.0"
+  },
+  "pnpm": {
+    "overrides": {
+      "esbuild": "0.19.2"
```

Comment:
We need to overwrite tsup esbuild version v18, since the sass plugin using v19 where the `Plugin` type definition changes. This can be removed after tsup updates to esbuild v19.